### PR TITLE
doc: document how to use 999-SNAPSHOT of quarkus

### PR DESCRIPTION
**Pull request by aureamunoz**
Related to #5687

### CODE REVIEW

**Comment by aureamunoz** [comment]
Hi, here the PR to doc how to be able to generate a project that uses locally built Quarkus. WDYT @geoand @maxandersen @aloubyansky ?

File: docs/src/main/asciidoc/maven-tooling.adoc [hunk 1]
```diff
@@ -40,6 +40,10 @@ The following table lists the attributes you can pass to the `create` command:
 | _mandatory_
 | The artifact id of the created project. Not passing it triggers the interactive mode.
 
+| `platformArtifactId`
+| `quarkus-universe-bom`
+| The artifact id of the of the target platform BOM. It should be `quarkus-bom` in order to use the locally built Quarkus.
```
Comment (aloubyansky) [inline docs/src/main/asciidoc/maven-tooling.adoc]: Please remove one `of the` in `The artifact id of the of the...`.

**Review by aloubyansky — CHANGES_REQUESTED** [review]
Thanks @aureamunoz There is a small misprint. Could you please fix it?
There actually are other relevant arguments.
`-DplatformGroupId` is the groupId of the target platform, the default value is `io.quarkus`, given that all the existing `platforms` are coming from `io.quarkus` this one won't practically be used explicitly. But it's still an option.
`-DplatformVersion` is the version of the platform you want the project to use. If it's not specified, the latest one will be resolved. It can also accept a version range, in which case the latest from the specified range will be used.
By default, the command will target the latest version of `quarkus-universe-bom` (unless specific coordinates have been specified). If you run offline however, it will look for the latest locally available. If `quarkus-universe-bom` (satisfying the default version range which is currently up to 2.0), it will fallback to the bundled platform based on `quarkus-bom` (the version will match the version of the plugin).
If you execute the command with

**Comment by aureamunoz** [comment]
@aloubyansky do you want also add this paragraphe to the doc? or it's just for clarification?
_By default, the command will target the latest version of quarkus-universe-bom (unless specific coordinates have been specified). If you run offline however, it will look for the latest locally available. If quarkus-universe-bom (satisfying the default version range which is currently up to 2.0), it will fallback to the bundled platform based on quarkus-bom (the version will match the version of the plugin).
If you execute the command with_

**Comment by aloubyansky** [comment]
I think it'd be good to mention it. Please correct this phrase
````
...available. If quarkus-universe-bom (satisfying the default version range which is currently up to 2.0),
````
to 
````
...available and if quarkus-universe-bom (satisfying the default version range which is currently up to 2.0) is not available locally,
````
Thank you!

**Review by aloubyansky — APPROVED** [review]

**Pull request merged**